dev/VSIX/extension.manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+    "$schema": "http://json.schemastore.org/vsix-publish",
+    "categories": [ "Build", "Coding", "Programming Languages" ],
```

Comment:
The existing marketplace entries currently have the categories set to 'Templates" and "Other'. Not sure what "Other" refers to but "Templates" should be here, and I'm not sure about "Build", "Coding", and "Programming Languages".